### PR TITLE
Tidy up code snippets

### PR DESCRIPTION
**Pull request by rsa33**
There's a handful of changes here related to fenced code blocks across the docs:

* 4-space indents have been removed (they appear verbatim in the rendered code)
* code block types have been set to `text` when missing (to avoid inconsistent non-highlight rendering)
* certain invalid keywords like `irc` have been substituted
* shell command blocks now use a CSS-positioned `$` so it doesn't get copied when selecting the block
  * shell session examples now use the `text` type (note that `bash-session` isn't available in our version of Hugo)
  * bash scripts use the `sh` type

This PR should cover my list of considerations in https://github.com/SRCF/docs/pull/47#issuecomment-1050833907.

### CODE REVIEW

File: content/reference/shell-and-files/scheduled-tasks.md [hunk 1]
```diff
@@ -125,19 +125,19 @@ when interacting with a group account's services (here using `foosoc`
...
-    socsudo () {
-
-    soc=\$1 shift sudo -Hu \$soc XDG\_RUNTIME\_DIR=/run/user/\$(id -u
-        \$soc) '\$@'
-    }
+```sh
+socsudo () {
+    soc=$1
+    shift
+    sudo -Hu soc XDG_RUNTIME_DIR=/run/user/$(id -u $soc) '$@'
```
Comment (edwinbalani) [inline content/reference/shell-and-files/scheduled-tasks.md:139]: `sudo -Hu $soc`

Comment (edwinbalani) [reply content/reference/shell-and-files/scheduled-tasks.md:139]: Probably want `"$@"`?

**Review by edwinbalani — CHANGES_REQUESTED** [review]
Thanks, looks very nice per the preview link you sent me!

There were a handful of inaccuracies/inconsistencies in the content, but I haven't flagged any of the ones you didn't touch with your changes -- so there's only one place to look at again.

If I understand properly, we're using the 'bash' highlighting language for a shell session and 'sh' for shell script snippets.

Also, what are the 'Go' and 'Markdown' types in the metadocs?  Do the capital letters mean anything special?

**Review by edwinbalani — APPROVED** [review]

**Comment by rsa33** [comment]
> If I understand properly, we're using the 'bash' highlighting language for a shell session and 'sh' for shell script snippets.

That is correct.

> Also, what are the 'Go' and 'Markdown' types in the metadocs?

I'm not sure why they're using Go currently (because Hugo is Go?) -- they're examples of snippets to be included in content files themselves, so Markdown seems the most appropriate choice, though it's bespoke syntax and doesn't really highlight usefully in any of them.

> Do the capital letters mean anything special?

It's case-insensitive as far as I can tell, so I've left the existing ones as-is to reduce the noise a bit.

**Comment by CMTC** [comment]
Perhaps it's a problem that is fine to ignore, but the css-trickery obviously only works in sufficiently clever browsers. Netsurf and the text-only browsers I've tried are not sufficiently clever.

**Comment by edwinbalani** [comment]
> Perhaps it's a problem that is fine to ignore, but the css-trickery obviously only works in sufficiently clever browsers.

So long as the behaviour in those browsers is for the prefix to just not appear, should we be OK?  Call it "progressive CSS" I suppose.

**Comment by edwinbalani** [comment]
> I'm not sure why they're using Go currently (because Hugo is Go?)

I think Hugo's templating language is really just Go templates.

**Comment by CMTC** [comment]
> So long as the behaviour in those browsers is for the prefix to just not appear, should we be OK? Call it "progressive CSS" I suppose.

That indeed seems to be the case. Surely since the symbol was previously shown, but is now not, we should regard it as regressive CSS...

**Comment by rsa33** [comment]
> I think Hugo's templating language is really just Go templates.

Upon closer inspection of the highlighters list, I think what we actually want here is `go-text-template` (which is supported in our version of Hugo).

**Comment by rsa33** [comment]
It has been noted that the CSS-applied `$` doesn't appear for each line of a multi-line command block, but that's an acceptable compromise for now.

**Pull request merged**